src/cim/components/Agent/AgentTable.tsx

```diff
@@ -16,20 +16,21 @@ import { usePagination } from '../../../common/components/hosts/usePagination';
 
 export const getAgentId = (agent: AgentK8sResource) => agent.metadata?.uid as string;
```

Comment:
I suggest not to make changes in the CIM code since you can't test it (I'm assuming you don't have ACM set up?) the CIM developers will fix it later

Reply:
Yes, I don't have ACM setup, so I'll revert the changes there. Thanks for pointing it out.